crates/turborepo-lib/src/lib.rs

```diff
@@ -62,38 +51,6 @@ pub fn get_version() -> &'static str {
         .trim_end()
 }
 
-pub fn main() -> Payload {
-    match shim::run() {
-        Ok(payload) => payload,
-        // We only print using miette for some errors because we want to keep
-        // compatibility with Go. When we've deleted the Go code we can
-        // move all errors to miette since it provides slightly nicer
-        // printing out of the box.
-        Err(
-            err @ (Error::MultipleCwd(..)
-            | Error::EmptyCwd { .. }
-            | Error::Cli(cli::Error::Run(run::Error::Builder(engine::BuilderError::Config(
-                config::Error::InvalidEnvPrefix { .. },
-            ))))
-            | Error::Cli(cli::Error::Run(run::Error::Config(
-                config::Error::TurboJsonParseError(_),
-            )))
-            | Error::Cli(cli::Error::Run(run::Error::Builder(BuilderError::Config(
-                config::Error::TurboJsonParseError(_),
-            ))))),
-        ) => {
-            println!("{:?}", Report::new(err));
-
-            Payload::Rust(Ok(1))
-        }
-        // We don't need to print "Turbo error" for Run errors
-        Err(err @ shim::Error::Cli(cli::Error::Run(_))) => Payload::Rust(Err(err)),
-        Err(err) => {
-            // This raw print matches the Go behavior, once we no longer care
-            // about matching formatting we should remove this.
-            println!("Turbo error: {err}");
-
-            Payload::Rust(Err(err))
-        }
-    }
+pub fn main() -> Result<i32, shim::Error> {
+    shim::run()
```

Comment:
I think we should avoid avoidable behavior changes in this PR and keep it to just deleting Go.